public_html/newjavascript.js

```diff
@@ -1,20 +1,26 @@
-function startTime()
-{
-var tm=new Date();
-var h=tm.getHours();
-var m=tm.getMinutes();
-var s=tm.getSeconds();
-m=checkTime(m);
-s=checkTime(s);
-document.getElementById('txt').innerHTML=h+":"+m+":"+s;
-t=setTimeout('startTime()',500);
-}
-function checkTime(i)
-{
-if (i<10)
-{
-i="0" + i;
-}
-return i;
-}
+class startTime {
+    constructor(){
+            this.count();
+    }
+    count(){
+            var date = new Date();
+
+            this.h = date.getHours();
+            this.m = date.getMinutes(); 
+            this.s = date.getSeconds();
+
+            this.m = this.checkTime(this.m); 
+            this.s = this.checkTime(this.s); 
 
+            document.getElementById('clock').innerHTML = this.h + ":" + this.m + ":" + this.s; 
+
+            this.t = setTimeout(this.count.bind(this), 500);
+    }
+
+    checkTime(i) {
+            if (i < 10) i = "0" + i;
+
+            return i;
+    }
+}
+var Clock = new startTime();
```

Comment:
kilka uwag odnośnie formatowania:
- wciecia powinny mieć 4 spacje,
- przed każdą metodą daj pustą linię
- w instrukcja if dajemy {} czyli 
if (...) {
    ...
}
- i na koniec taka pierdoła, dajemy spacje międzi ) { przy nazwie metody

Reply:
jeszcze kwestia inicjowania klasy, fajnie jakby było to zrobione w pliku html